src/main.c

```diff
@@ -593,6 +592,18 @@ BOOL save_all_packs() {
 	return success;
 }
 
+BOOL validate_playable() {
```

Comment:
```suggestion
static BOOL validate_playable(void) {
```
The parameter list thing doesn't really fit the style of the rest of the file, but since there's also no declaration in ebmusv2.h, it's still a good idea to add the proper parameter list here, right?